src/olympia/search/filters.py

```diff
@@ -189,6 +189,32 @@ def get_es_filter(self):
         return [Q('term', tags=tag) for tag in self.get_value()]
 
 
+class AddonFeaturedFilterParam(AddonFilterParam):
+    query_param = 'featured'
+    reverse_dict = {'': True, 'true': True}
+    valid_values = [True]
+    es_field = 'featured_for'
+
+    def get_es_filter(self):
+        self.get_value()  # Call to validate the value - we only want True.
+        app_filter = AddonAppFilterParam(self.request)
+        app = (app_filter.get_value()
+               if self.request.GET.get(app_filter.query_param) else None)
+        locale = self.request.GET.get('locale')
```

Comment:
`locale` ? Can't we re-use `lang` ?

Reply:
We should indeed use `lang`; we don't send a `locale` query param from the frontend.

Reply:
yeah, it was supposed to be `lang`.  Though... we are overloading the purpose of the `lang` parameter - what if we want addons featured in `fr` but want all the localisations to be sent (or want a different localisation)?

Reply:
https://github.com/mozilla/addons-server/pull/6339/commits/e91a44bf3179d7c4a3f1e05fbc6e0d515458495b